pkg/v1/tkg/tkgctl/client.go

```diff
@@ -277,3 +280,11 @@ func ensureConfigImages(configDir string, tkgConfigUpdater tkgconfigupdater.Clie
 
 	return tkgConfigUpdater.EnsureConfigImages()
 }
+
+func getFeatureFlagClient() client.FeatureFlagClient {
+	featureFlagClient, err := config.GetClientConfig()
+	if err != nil {
+		featureFlagClient = &configv1alpha1.ClientConfig{}
```

Comment:
I don't know the context here, but I'm concerned about failing silently if there's an error getting the config. Is there any way to surface a warning as to what this error was?

Reply:
We thought that it would only get here if the file containing the feature flags didn't exist, so that would mean there are no feature flags to worry about. Though we moved this code into the AllClients bit and now return an error which feels like a safer choice. Thanks for the suggestion!